docs/system-admin-guide/working-days/README.md

```diff
@@ -17,9 +17,17 @@ To change this setting, navigate to *Administration* → *Working days*.
 
 To change this setting, unselect days that you would like to define as non-working, and select ones that you would like to consider working, and click on **Save**.
 
+### Specific non-working days
+
+You can also designate specific dates (such as national or local holidays) as non-working days. Work packages in regular scheduling mode will skip these days at they will not be included when calculating duration.
```

Comment:
@psatyal do we want to emphasise here also that this non-working days work for all the instance and it is not individual?

Reply:
Hi @marcalcobe. Yes, this is explained in the paragraphe below titled "effect on scheduling":

> As an instance-level setting, any change here will affect the scheduling of all work packages in all projects in that instance. It is currently not possible to define working days at a project-level.